File: pset1/cash/cash.c
#include <stdio.h>
#include <math.h>
#include <cs50.h>

int main(void)
{
    float totalChange;

    do 
    {
        totalChange = get_float("How much change is owed?: \n");
    } while (totalChange < 0);
    
    
    int cents = round(totalChange * 100);
    int coins = 0;
    
   while (cents >= 25) {
       cents -= 25;
       coins++;
   } while (cents >= 10) {
       cents -= 10;
       coins++;
   } while (cents >= 5) {
       cents -= 5;
       coins++;
   } while (cents >= 1) {
       cents -= 1;
       coins++;
   }

    printf("%i\n",coins);
}

File: pset6/cash/cash.py
import cs50
import math

while True:
    totalChange = cs50.get_float("Change Due ($) : ")
    if totalChange > 0:
        break
    
cents = totalChange * 100
coins = 0

while cents > 0:
    while cents >= 25:
        cents -= 25
        coins += 1
    while cents >= 10:
        cents -= 10
        coins += 1
    while cents >= 5:
        cents -= 5
        coins += 1
    while cents >= 1:
        cents -= 1
        coins += 1
        
print(coins)

